hackathon_site/event/jinja2/event/reset_password/reset_password_email.html

```diff
@@ -0,0 +1,17 @@
+<p>Hello {{ user.get_username() }},</p>
+
+<p>You're receiving this email because you requested a password reset for your {{ event_name }} user account.</p>
+
+<p>Please go to the following page and choose a new password:</p>
+
+<a href="{{ protocol }}://{{ domain }}{{ url('event:reset_password_confirm', args=[uid,token])}}">Reset Password</a>
```

Comment:
Created an issue in the newhacks repo to remind us to test this: https://github.com/ieeeuoft/newhacks-site-2020/issues/22